$NetBSD: patch-src_CMakeLists.txt,v 1.2 2024/07/12 11:13:13 micha Exp $

Disable downloads for Apple platforms.
Remove" -Werror".
Add "-Wno-cast-function-type".
Remove duplicated flags (GCC fails because they are inserted with quotes).

--- src/CMakeLists.txt.orig	2024-07-09 18:15:50.000000000 +0000
+++ src/CMakeLists.txt
@@ -237,7 +237,7 @@ else()
     target_compile_definitions(eurekasrc PUBLIC NO_OPENGL)
 endif()
 
-if(APPLE OR WIN32)
+if(WIN32)
     message(STATUS "Using local FLTK for portability.")
 
 	set(LOCAL_FLTK_VERSION "1.3.8")
@@ -359,7 +359,7 @@ if(MSVC)
 endif()
 
 if ( CMAKE_CXX_COMPILER_ID MATCHES "Clang" OR CMAKE_CXX_COMPILER_ID STREQUAL "GNU" )
-    target_compile_options(eurekasrc PUBLIC "${CMAKE_CXX_FLAGS}" -Wall -Wextra -Werror
+    target_compile_options(eurekasrc PUBLIC -Wall -Wextra -Wno-cast-function-type
                            -Wno-unused-parameter -Wno-missing-field-initializers -Wunused-variable)
 endif()
 if ( MSVC )
